common/src/main/java/org/conscrypt/OpenSSLCipherRSA.java

```diff
@@ -166,6 +166,10 @@ protected void doCryptoInit(AlgorithmParameterSpec spec)
 
     protected void engineInitInternal(int opmode, Key key, AlgorithmParameterSpec spec)
             throws InvalidKeyException, InvalidAlgorithmParameterException {
+        if (null == key) {
+            throw new IllegalArgumentException("Key must not be null");
```

Comment:
This seems like `InvalidKeyException` is appropriate here instead of an unchecked `IllegalArgumentException`.